<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grade_Calculator</class>
 <widget class="QMainWindow" name="Grade_Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grade_Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Hw2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw2scroll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw2spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Hw1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw1scroll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw1spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Hw3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw3scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw3spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Hw4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw4scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw4spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Hw5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw5scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw5spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Hw6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw6scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw6spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Hw7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw7scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw7spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hw8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="hw7scroll_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw8spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Midterm 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="m1scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m1spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>150</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Midterm 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="m2scroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m2spin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>125</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="fscroll">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fspin"/>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>hw1scroll</zorder>
   <zorder>hw1spin</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>hw3scroll</zorder>
   <zorder>hw3spin</zorder>
   <zorder>hw4scroll</zorder>
   <zorder>hw4spin</zorder>
   <zorder>hw5scroll</zorder>
   <zorder>hw5spin</zorder>
   <zorder>label_3</zorder>
   <zorder>hw6scroll</zorder>
   <zorder>hw7scroll</zorder>
   <zorder>hw7scroll_2</zorder>
   <zorder>hw6spin</zorder>
   <zorder>hw7spin</zorder>
   <zorder>hw8spin</zorder>
   <zorder>Calculate</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>m1scroll</zorder>
   <zorder>m1spin</zorder>
   <zorder>m2scroll</zorder>
   <zorder>m2spin</zorder>
   <zorder>fscroll</zorder>
   <zorder>fspin</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>hw1scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw1spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7scroll_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw8spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw8spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7scroll_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m1scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m1spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m1spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m1scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>577</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m2scroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m2spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m2spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m2scroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fscroll</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fspin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fspin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fscroll</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
